app/util.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timedelta
+
+import boto3
+import pytz as pytz
+
+TODAY = datetime.now(pytz.timezone('US/Pacific'))
+TODAY_YYYY_MM_DD = TODAY.strftime('%Y-%m-%d')
+TODAY_IS_WEEKEND = TODAY.weekday() >= 4  # Days are 0-6. 4=Friday, 5=Saturday, 6=Sunday, 0=Monday
+MIN_TERMINATION_WARNING_YYYY_MM_DD = (TODAY - timedelta(days=3)).strftime('%Y-%m-%d')
+
+
+def money_to_string(amount):
+    return '${:.2f}'.format(amount)
+
+
+def quote(value):
+    return '"' + value + '"'
+
+
+def make_tags_dict(tags_list: list) -> dict:
+    tags = dict()
+    for tag in tags_list:
+        tags[tag['Key']] = tag['Value']
+    return tags
```

Comment:
This could actually use a dictionary comprehension

```suggestion
    return { tag['Key']: tag['Value'] for tag in tags_list }
```